### DISCUSSION TITLE
Persist link(ed)-files

### DISCUSSION THREAD

**Discussion by mhoran** [blinksh/blink#2077] [Ideas]
It should be possible to persist `link-files` as of iOS 13. See https://developer.apple.com/documentation/uikit/view_controllers/providing_access_to_directories?language=objc:
>  Your app can even save a bookmark for this URL, letting it access the directory the next time it launches.
There is sample code for this scenario.

This is implemented in iSH, where `mount` persists across app termination and restart.

**Comment by carloscabanero** [comment]
Yes it is. And now is the perfect time to finally do this. Mounting the new remote provider makes use of the same functions. I had it implemented to automatically mount them in a remote, but it makes more sense to make it selectable and we can make other apps part of it too. I’m pushing the next build on TF today. We should have this functionality and cut a release version next week!

**Reply by carloscabanero** [reply]
This is now supported in 18.1.0 build 963!

**Reply by mhoran** [reply]
Works great, thanks!
